app/assets/stylesheets/src/shared/_selectize.scss

```diff
@@ -3,24 +3,6 @@
 // https://github.com/selectize/selectize.js/issues/1614.
 @import "selectize/dist/css/selectize.bootstrap5";
 
-.selectize-input {
-  input[placeholder] {
-    width: inherit !important;
-  }
-}
```

Comment:
was added [here](https://github.com/toughbyte/hub/pull/6343) and seems became unnecessary because selectize automatically calculates input width depending on placeholder text
![image](https://github.com/user-attachments/assets/4d432165-41c3-4fa6-9e29-4b9d3df42bc9)